releases/in-progress.html

```diff
@@ -9,16 +9,17 @@
 
   <div class="version">
     <a name="latest"></a>
-    <a name="15.2"></a>
-    <h4>15.2 (beta)</h4>
+    <a name="15.3"></a>
+    <h4>15.3 (beta)</h4>
 
     <div class="section">
-      <h5>Improvements to the console log</h5>
+      <h5>Programming the flux capacitor</h5>
```

Comment:
no. i know its just a place holder. but a small description could have sufficed i guess :D anyway looks good to me. will merge once its ready.